packages/terra-menu/src/MenuItem.jsx

```diff
@@ -301,12 +300,12 @@ class MenuItem extends React.Component {
 
     const screenReaderResponse = (
       <>
-        { MenuUtils.isMac() && <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.menu.index' }, { index: itemIndex + 1, totalItems })} /> }
+        { MenuUtils.isMac() && <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.menu.index' }, { index: index + 1, totalItems })} /> }
         { MenuUtils.isMac() && (isGroupItem || toggleable) && <VisuallyHiddenText text={markAsToggled ? intl.formatMessage({ id: 'Terra.menu.selected' }) : intl.formatMessage({ id: 'Terra.menu.unselected' })} /> }
         {/* Adds context for item with submenu-items */}
         { subMenuItems.length > 0 && <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.menu.itemWithSubmenu' })} /> }
         {/* Adds context for navigating back to parent menu from submenu */}
-        { this.itemNode && this.itemNode.parentNode.getAttribute('data-submenu') === 'true'
+        { this.itemNode && this.itemNode.parentNode.getAttribute('data-submenu') === 'true' && index === 0
```

Comment:
why do we need index === 0 check here ?

Reply:
This was added to retain this instruction only to the first item in submenu. This was requested to be removed for all other items.